test/amoc_api_scenario_handler_SUITE.erl

```diff
@@ -177,45 +170,37 @@ get_url() ->
 
 -spec get_request(string()) -> 
     {integer(), jiffy:jiffy_decode_result()}.
-get_request(URL) ->
-    Header = [],
-    HTTPOpts = [],
-    Opts = [],
-    Result = httpc:request(get,
-                           {URL, Header},
-                           HTTPOpts,
-                           Opts),
-
-    {ok, {{_HttpVsn, CodeHttp, _Status}, _, Body}} = Result,
-    BodyErl = case Body of
-                  [] -> 
-                      [];
-                  _ ->
-                      jiffy:decode(erlang:list_to_bitstring(Body))
-              end,
-    {CodeHttp, BodyErl}.
+get_request(Path) ->
+    request(get, Path).
 
 -spec patch_request(string(), string()) ->
     {integer(), jiffy:jiffy_decode_result()}.
-patch_request(URL, RequestBody) ->
-    Header = "",
-    Type = "application/json",
-    HTTPOpts = [],
-    Opts = [],
-    Result = httpc:request(patch,
-                           {URL, Header, Type, RequestBody},
-                           HTTPOpts,
-                           Opts),
-    {ok, {{_HttpVsn, CodeHttp, _Status},_, Body}} = Result, 
+patch_request(Path, RequestBody) ->
+    request(<<"PATCH">>, Path, RequestBody).
```

Comment:
Why here the HTTP Verb is passed as binary while in the line 173 as atom? That should be unified.